php/php.dbgp/src/org/netbeans/modules/php/dbgp/breakpoints/ExceptionBreakpoint.java

```diff
@@ -27,6 +30,12 @@
  */
 public class ExceptionBreakpoint extends AbstractBreakpoint {
     private final String exceptionName;
+    @NullAllowed
+    private String message;
+    @NullAllowed
+    private String code;
+    private static final String FONT_GRAY_COLOR = "<font color=\"#999999\">"; //NOI18N
+    private static final String CLOSE_FONT = "</font>"; //NOI18N
```

Comment:
Please, could you put constants before class fields?

Reply:
Done.